features/append/append.feature

```diff
@@ -1,11 +1,11 @@
 Feature: append a new feature branch to an existing feature branch
 
   Background:
-    Given my repo has a feature branch "existing"
+    Given a feature branch "existing"
     And the commits
       | BRANCH   | LOCATION      | MESSAGE         |
       | existing | local, origin | existing commit |
-    And I am on the "existing" branch
+    And the current branch is "existing"
```

Comment:
Humans aren't on Git branches. Humans can run Git commands to check out Git branches, or make them the currently checked out branch.